lib/regexpcache/regexpcache.go

```diff
@@ -0,0 +1,201 @@
+package regexpcache
```

Comment:
I'd rename this cache from `regexpcache` to `lrucache`, since it is a general-purpose LRU cache, which can be used for caching any objects, not only regexps. This cache can be used in other places at VictoriaMetrics codebase, which now use inline map-based cache implementation.

Reply:
renamed